### PR TITLE
#1 : fix PHP 7.2 warning + light refactoring

### PR DESCRIPTION
**Pull request by herewithme**

### CODE REVIEW

**Review by romain-d — APPROVED** [review]

**Comment by Rahe** [comment]
Why removing the constant for the option ?

File: bea-missed-schedule.php [hunk 1]
```diff
@@ -1,42 +1,59 @@
...
+	 *
+	 * @return bool
+	 */
 	public static function publish_missed_schedule() {
 		global $wpdb;
 
 		$scheduled_ids = $wpdb->get_col( "SELECT `ID` FROM `{$wpdb->posts}` WHERE( ((`post_date`>0)&&(`post_date`<=CURRENT_TIMESTAMP())) OR ((`post_date_gmt`>0)&&(`post_date_gmt`<=UTC_TIMESTAMP())) )AND `post_status` = 'future' LIMIT 0, 10" );
-		if ( false == $scheduled_ids ) {
+		if ( false === $scheduled_ids || empty( $scheduled_ids ) ) {
 			return false;
```
Comment (Rahe) [inline bea-missed-schedule.php:57]: If no return true, just return will be enough.

**Comment by herewithme** [comment]
@Rahe The constant brings nothing for 3 uses, it complicates at most the readability of the code.

File: bea-missed-schedule.php (hunk 1 above)
Comment (herewithme) [reply bea-missed-schedule.php:57]: Not understood.

**Review by ghost — APPROVED** [review]
but I agree with NJ on the fact that "why do you change constant for string ?", even if we never change that why do you change that ? It's cool to have all vars at the top of the file for readability.

**Comment by herewithme** [comment]
@TweetPressFr @Rahe OK, I admit that it doesn't change much. but is that a problem in this case? a 50-line plugin? :)

File: bea-missed-schedule.php (hunk 1 above)
Comment (Rahe) [reply bea-missed-schedule.php:57]: if there is no `return true` at the end of the method, then the `return false` isn't quite necessary. just `return` will do the job.

**Comment by ghost** [comment]
@herewithme there is no reason to decline it because the PHP 7.2 fix is useful but the light refactoring part is useless imho.

**Comment by herewithme** [comment]
Yes, that's what I understood from your feedback: https://github.com/BeAPI/bea-missed-schedule/commit/ef2b77f87426664b92013b297e4b73cb843e6e19

**Pull request closed without merging**